install_and_configure.yml

```diff
@@ -0,0 +1,121 @@
+---
+ - hosts: epidose
+   gather_facts: false
+   become: yes
+   vars:
+     ssh_key: '~/.ssh/id_rsa.pub'
+
+   tasks:
+    - name: create 'testgroup' group
+      tags: production, development
+      group:
+        name: testgroup
+        state: present
+
+    - name: allow 'testgroup' group to have passwordless sudo
+      tags: production, development
+      lineinfile:
+        dest: /etc/sudoers
+        state: present
+        regexp: '^%testgroup'
+        line: '%testgroup ALL=(ALL) NOPASSWD: ALL'
+        validate: 'visudo -cf %s'
+  
+    - name: add sudo user to 'testgroup' group
+      tags: production, development
+      user:
+        name: epidose
+        groups: testgroup
+        append: yes
+        state: present
+        createhome: yes
+
+    - name: deploy ssh key
+      tags: production, development
+      authorized_key:
+        key: "{{ lookup('file', ssh_key) }}"
+        user: epidose
+
+    - name: install libraries and packages
+      tags: production, development
+      become: yes
+      become_user: epidose
+      apt:
+        name: ['git', 'libbluetooth-dev', 'libglib2.0-dev', 'python3-dev', 'python3-setuptools', 'shellcheck', 'sqlite3', 'virtualenv', 'dh-virtualenv', 'debhelper', 'supervisor']
+
+    - name: setup production environment
+      tags: production
+      become: yes
+      become_user: epidose
+      shell: |
+       cd /home/epidose
+       git clone https://github.com/dspinellis/epidose
+       cd epidose      
+       virtualenv venv -p /usr/bin/python3
+       . venv/bin/activate
+       pip3 install -e ".[dev,test,deploy]"
+       make package
+       sudo make install
+      register: setup_env_output
+
+    - name: setup development environment
+      tags: development
+      become: yes
+      become_user: epidose
+      shell: |
+       cd /home/epidose
+       git clone https://github.com/dspinellis/epidose
```

Comment:
It's `eellak/epidose` now

Reply:
Ok!

Reply:
1) I guess by the vanilla distribution you are referring to this one: Raspberry Pi OS (32-bit) Lite Minimal image based on Debian Buster right?
2)  By removing the known password you mean just the pi user? I am asking because during a Skype call you told me to leave the pi user there since someone may use it.
3) For installation instructions, should I remove all the installation-related sections after I add the _How to install with Ansbile_ or should I leave them there too?
4) Could you please share a link about the maintenance shell. I cannot find anything related to it.

Reply:
1. Yes
2. I think it's safer to remove it and install a new one
3. We may have two sections: One for creating an image, and one for installing on an existing system
4. I mean allowing the `epidose` user to login via private key and run `sudo`

Reply:
OK, just in case, I have tested it two times on a vanilla Raspbian distribution.
Let me know what changes are still required.